packages/x-charts/src/LineChart/LineChart.tsx

```diff
@@ -124,6 +124,10 @@ export interface LineChartProps
    * @default false
    */
   skipAnimation?: boolean;
+  /**
+   * If `true` marks will render `<circle />` instead of `<path />` and drop theme override for faster rendering.
+   */
+  markPerfUpdate?: boolean;
```

Comment:
```suggestion
  usePerformantMarks?: boolean;
```

Reply:
```suggestion
  improveMarkPerformance?: boolean;
```

Reply:
`markPerfUpdate` sounds vague, "update" is a status, so it doesn't fully describe the action I think.

Reply:
What about `experimentalMarkRendering` To say it's planned to be the future default behavior in v8?

Reply:
Yeah, I think that would be great